### PR TITLE
Header footer resume check in

### PR DESCRIPTION
**Pull request by asorichetti**
Think I may have an issue as my footer is shooting back the error that "'footerRef.current' is possibly 'null'."

### CODE REVIEW

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        if (iosTopBarHandler){
+            window.addEventListener('scroll', handleScrollios);
+        }
+        handleScrollios();
+        return function cleanup(){
+            if (iosTopBarHandler){
+                window.removeEventListener('scroll', handleScrollios);
+            }
+        };
+    });
```
Comment (hweeks) [inline src/components/Footer/index.tsx:28]: is this still needed in modern iOS? it is rougnly 4 year old code

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    return(
+        <Footer>
+            <FootText ref={footerRef}>
+                <FootLink href="www.linkedin.com/in/alex-sorichetti">LinkedIn</FootLink>
+                -
```
Comment (hweeks) [inline src/components/Footer/index.tsx:35]: this should be encoded as an html entity

File: src/components/Footer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                <ResumeHome></ResumeHome>
+                -
```
Comment (hweeks) [inline src/components/Footer/index.tsx:37]: same here

File: src/components/Footer/styles.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    align-items: center;
+    padding: 1.5rem;
+    justify-content: center;
+`;
+
+export const FootText = styled.p`
+    text-align: center;
+    font-size: 3rem;
+    -webkit-text-fill-color: #F5D547;
+    `;
```
Comment (hweeks) [inline src/components/Footer/styles.ts:15]: can you make the spacing and line ends consistent? 9 and 15/23 are different. also a new line at the bottom of a file is standard, you can make your editor do this automaitcally

File: src/components/Footer/styles.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+export const FootLink = styled.a`
+    font-size: 3rem;
+    -webkit-text-fill-color: #F5D547;
+    @media (max-width: 768px){
+        font-size: 12vw;
```
Comment (hweeks) [inline src/components/Footer/styles.ts]: is there a reason to lock font size to viewport width?

File: src/components/resume/index.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+import { FootText, FootLink } from '../Footer/styles';
+import resumePdf from '../../../static/resume.pdf';
+import { IfOpenTrue, StyledLayout } from './styles';
+
+export const ResumeHome = () => {
+    const [open, setOpen] = useState(false);
+    const toggleOpen = () => setOpen(!open);
+    return(
+        <StyledLayout>
+            <IfOpenTrue open={!open}>
```
Comment (hweeks) [inline src/components/resume/index.tsx]: if there is nothing in a tag, you can self end it

```
<IfOpenTrue open={!open} />
```

File: src/components/resume/index.tsx [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+            </IfOpenTrue>
+        
+        <IfOpenTrue open={open}>
+        <div>
```
Comment (hweeks) [inline src/components/resume/index.tsx]: why the extra div around the div you return?

File: src/components/resume/styles.ts [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+export const IfOpenTrue = styled.div<{open: boolean}>`
+    display: ${(props) => (props.open ? 'block':'none')}
+    `;
+
+export const StyledLayout = styled.span`
+    display: inline-block;
+    text-decoration: underline;
+    text-decoration: overline;
+    cursor: pointer
+    `;
```
Comment (hweeks) [inline src/components/resume/styles.ts]: can we fix spacing and newline here?

File: src/index.tsx [hunk 9]
```diff
@@ -1,10 +1,40 @@
-import React from "react";
+import * as React from "react";
```
Comment (hweeks) [inline src/index.tsx]: this style of import should ne be needed and indicates following an older react guideline. why was this changed?

File: src/index.tsx [hunk 10]
```diff
@@ -1,10 +1,40 @@
...
+    backgroundColor: '#334445',
+    color: '#F5D547',
+  },
+};
 const HomePage = () => {
-  return <div>
-    Home
+
+return( 
+  <div style={styles.container}>
```
Comment (hweeks) [inline src/index.tsx:33]: please stick to styled-components, mixing two types of styling makes it hard to reason about how stlying is handled

File: static/index-template.html [hunk 11]
```diff
@@ -7,6 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>
     <%= htmlWebpackPlugin.options.title %>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
```
Comment (hweeks) [inline static/index-template.html]: a link is not a valid child of the title, it must be in the head

File: tsconfig.json [hunk 12]
```diff
@@ -11,5 +11,5 @@
   "include": [
     "media.d.ts",
     "src/**/*"
-  ]
+, "src/components/resume"  ]
```
Comment (hweeks) [inline tsconfig.json]: why have you added a single file? this is handeld by the `src/**/*` import

**Review by hweeks — CHANGES_REQUESTED** [review]
a series of styling and approach updates are needed

File: tsconfig.json (hunk 12 above)
Comment (asorichetti) [reply tsconfig.json]: I didn't recall adding this, but it has been removed after your note

File: src/index.tsx (hunk 9 above)
Comment (asorichetti) [reply src/index.tsx]: Similar to the resume import, this is a auto-complete and auto-correction suggestion that went wrong, has since been deleted

File: src/components/resume/styles.ts (hunk 8 above)
Comment (asorichetti) [reply src/components/resume/styles.ts]: Corrected indents and added new line to the bottom

File: src/components/Footer/styles.ts (hunk 5 above)
Comment (asorichetti) [reply src/components/Footer/styles.ts]: Changed this to rem rather than vw as there was no valid reason for this.

File: src/components/Footer/styles.ts (hunk 4 above)
Comment (asorichetti) [reply src/components/Footer/styles.ts:15]: Fixed with the new line as well as consistent tab indents

File: src/components/Footer/index.tsx (hunk 1 above)
Comment (asorichetti) [reply src/components/Footer/index.tsx:28]: Went with this after viewing my own mobile device browser, will look into documentation to be sure that it is still a necessity

File: src/components/resume/index.tsx (hunk 6 above)
Comment (asorichetti) [reply src/components/resume/index.tsx]: Will fix up to this point

File: static/index-template.html (hunk 11 above)
Comment (asorichetti) [reply static/index-template.html]: removed from the title and placed within the head

File: src/index.tsx (hunk 10 above)
Comment (asorichetti) [reply src/index.tsx:33]: styles.container is referencing the const that is described which sets the background fill, as well as setting text padding, and colour. Haven't yet changed this part, but I will double-check styled styled-components for if there is a built-in edit for the background colour and making sure the back fill goes edge to edge.

File: src/components/resume/index.tsx (hunk 6 above)
Comment (asorichetti) [reply src/components/resume/index.tsx]: Fixed formatting here

File: src/components/Footer/index.tsx (hunk 2 above)
Comment (asorichetti) [reply src/components/Footer/index.tsx:35]: FootLink is an <a> assigned variable, but I do not currently know if I need to tweak this any farther for it to be working.

Comment (asorichetti) [reply src/components/Footer/index.tsx:35]: Added in opening "&" and closing ";" around text to encode as HTML elements

**Review by hweeks — APPROVED** [review]
